R/defaults.R

```diff
@@ -3,7 +3,6 @@
 #' @inheritParams epidist_validate
 #' @param ... Additional arguments passed to method.
 #' @family defaults
-#' @importFrom cli cli_abort
```

Comment:
the common CLI functions are good examples of things we might want in the namespace as we use them a lot

Reply:
Yep fine with adding them

Reply:
https://github.com/epinowcast/epidist/issues/341